crates/client/src/client/schema/tx/transparent_tx.rs

```diff
@@ -80,13 +82,21 @@ pub struct Transaction {
     pub id: TransactionId,
     pub tx_pointer: Option<TxPointer>,
     pub input_asset_ids: Option<Vec<AssetId>>,
+    /// The list of all contracts from the inputs of the transaction.
+    ///
+    /// It is a helper function and doesn't have a corresponding field in the transaction.
```

Comment:
What is the "helper function"? This is a field on a struct.

Reply:
Gotcha. The function isn't public though, but this doc comment is public, so maybe mentioning the function isn't necessary as long as you say it's generated rather than part of the original `Transaction`.